### PR TITLE
Added PR template - Issue (#4)

### PR DESCRIPTION
**Pull request by sabarish98**
Hi @BALaka-18 

I added the PR template for the repository. Please go through the template and let me know if there are any changes needed.

Issue : #4 

Thanks

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- [ ] Adding tests to existing code
+- [ ] New Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation Update
+
+## Description of the change
+
+## Related Issues & Documents (if needed)
+
```
Comment (BALaka-18) [inline .github/PULL_REQUEST_TEMPLATE.md]: The above two headings will come before  ```## What type of PR is this? (check all applicable)```

**Review by BALaka-18 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+_Please replace this line with instructions on how to test your changes
+
+## Added tests?
+
+- [ ] Yes
+- [ ] No, because they aren't needed
+- [ ] No, because I need help
+
+## Added to documentation / readme?
```
Comment (BALaka-18) [inline .github/PULL_REQUEST_TEMPLATE.md]: Added to documentation / readme / updated ```requirements.txt```(if needed) ?

**Review by BALaka-18 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (sabarish98) [reply .github/PULL_REQUEST_TEMPLATE.md]: Made the changes, please check !

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (sabarish98) [reply .github/PULL_REQUEST_TEMPLATE.md]: Made the changes, please check !

**Review by BALaka-18 — APPROVED** [review]

**Pull request merged**